simulator/risc_v/t/unit_test.cpp

```diff
@@ -547,6 +547,48 @@ TEST_CASE("RISCV RV64 ror")
     }
 }
 
+TEST_CASE("RISCV RV32 rori")
+{
+    // |  01100  |     imm     |   rs1   | 101 |    rd   |   0010011   |  RORI
+    CHECK( RISCVInstr<uint32>( 0x6007D793).get_disasm() == "rori $a5, $a5, 0"); // 01100 | 0000000 (0) | 01111 ($a5) | 101 | 01111 ($a5) | 0010011
+    TestData<uint32>test_0 = TestData<uint32>( 0x01234567, 0,  0x01234567); // src2 (0) is not used
+    INFO( "Iteration: " << 0);
+    test_0.make_test ("rori", 0);
+    CHECK( RISCVInstr<uint32>( 0x6047D793).get_disasm() == "rori $a5, $a5, 4"); // 01100 | 0000100 (4) | 01111 ($a5) | 101 | 01111 ($a5) | 0010011
+    TestData<uint32>test_4 = TestData<uint32>( 0x01234567, 0,  0x70123456); // src2 (0) is not used
+    INFO( "Iteration: " << 1);
+    test_4.make_test ("rori", 4);
+    CHECK( RISCVInstr<uint32>( 0x6087D793).get_disasm() == "rori $a5, $a5, 8"); // 01100 | 0001000 (8) | 01111 ($a5) | 101 | 01111 ($a5) | 0010011
+    TestData<uint32>test_8 = TestData<uint32>( 0x01234567, 0,  0x67012345); // src2 (0) is not used
+    INFO( "Iteration: " << 2);
+    test_8.make_test ("rori", 8);
+    CHECK( RISCVInstr<uint32>( 0x61F7D793).get_disasm() == "rori $a5, $a5, 31"); // 01100 | 0100000 (32) | 01111 ($a5) | 101 | 01111 ($a5) | 0010011
+    TestData<uint32>test_31 = TestData<uint32>( 0x01234567, 0,  0x02468ace); // src2 (0) is not used
+    INFO( "Iteration: " << 3);
+    test_31.make_test ("rori", 31);
+}
+
+TEST_CASE("RISCV RV64 rori")
+{
+    // |  01100  |  imm(shamt)  |   rs1   | 101 |    rd   |   0010011   |  RORI
+    CHECK( RISCVInstr<uint64>( 0x6007D793).get_disasm() == "rori $a5, $a5, 0"); // 01100 | 0000000 (0) | 01111 ($a5) | 101 | 01111 ($a5) | 0010011
+    TestData<uint64>test_0 = TestData<uint64>( 0x0123'4567'89ab'cdef, 0,  0x0123'4567'89ab'cdef); // src2 (0) is not used
+    INFO( "Iteration: " << 0);
+    test_0.make_test ("rori", 0);
+    CHECK( RISCVInstr<uint64>( 0x6047D793).get_disasm() == "rori $a5, $a5, 4"); // 01100 | 0000100 (4) | 01111 ($a5) | 101 | 01111 ($a5) | 0010011
+    TestData<uint64>test_4 = TestData<uint64>( 0x0123'4567'89ab'cdef, 0,  0xf012'3456'789a'bcde); // src2 (0) is not used
+    INFO( "Iteration: " << 1);
+    test_4.make_test ("rori", 4);
+    CHECK( RISCVInstr<uint64>( 0x6087D793).get_disasm() == "rori $a5, $a5, 8"); // 01100 | 0001000 (8) | 01111 ($a5) | 101 | 01111 ($a5) | 0010011
```

Comment:
Not sure there is much value in testing RV64 disassembly once we test it for RV32 (there is no extra code to cover)